workers/unity/Packages/com.improbable.gdk.mobile/Android/Utility/DeviceInfo.cs

```diff
@@ -0,0 +1,30 @@
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile.Android
+{
+    public class DeviceInfo
```

Comment:
This can be `static`

Reply:
Would also mildly rather change the name of the type than just the namespace. It's not immediately obvious in code which `DeviceInfo` you are looking at.